CHANGELOG.md

```diff
@@ -1,91 +1,96 @@
 # Change Log
 
+## [0.4.3] (October 10, 2023)
+
++ Minor fixes to common_filter.py #659
++ Fixes to dj.config #658
+
 ## [0.4.2] (October 10, 2023)
 
 ### Infrastructure / Support
 
-- Bumped Python version to 3.9. #583
-- Updated user management helper scripts for MySQL 8. #650
-- Centralized config/path handling to permit setting via datajoint config. #593
-- Fixed Merge Table deletes: error specificity and transaction context. #617
++ Bumped Python version to 3.9. #583
```

Comment:
Is the change to `+` prescribed by a standard. I have `markdownlint` on my IDE which changes bullets to `-`. Would hate have us go back and forth on diff standards